routemodel/tests/test_speedlimits.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import os.path
+sys.path.append(os.path.dirname(sys.path[0]))
+
+import pandas as pd
+import json
+import data_retrieval.speedlimits as spdlts
+
+def test_speedlimits_points_builder_correct_inputs():
+    list_of_points = [{"45": "-74"}, {"45.5": "-74.5"}, {"46": "-75"}]
+    points = spdlts.speedlimits_points_builder(list_of_points)
+    check = "45,-74;45.5,-74.5;46,-75"
+    assert(points == check)
+        
+def test_speedlimits_points_builder_no_inputs():
+    list_of_points  = []
+    points = spdlts.speedlimits_points_builder(list_of_points)
+    check = ""
+    assert(points == check)
+    
+def test_format_speedlimits_query_valid_points_default_inputs():
+    list_of_points = [{"45": "-74"}, {"45.5": "-74.5"}, {"46": "-75"}]
+    points = spdlts.speedlimits_points_builder(list_of_points)
+    query = spdlts.format_speedlimits_query(points)
+    check = "Routes/SnapToRoad?includeSpeedLimit=true&travelMode=driving&" + \
+            "points={}&speedUnit={}".format(points,
+                                            'KPH')
```

Comment:
Don't think these are worth their own lines

Reply:
I thought a long string would be annoying to try to read so

there are easier ways to split these i guess, or i can put it all on one line

Reply:
Sorry not all of them, just 26 and 27 can be combined